src/main/resources/config/liquibase/changelog/20210605180001_changelog.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="hashamov" id="20210605180001">
+        <addColumn tableName="grade_step">
+            <column name="lower_bound_points" type="DOUBLE"/>
```

Comment:
please delete those two lines, so that in total only `course.max_points` will be added

Reply:
I would have. However, since the changes were already executed on multiple test servers, I created another one, so there are no obsolete columns. Just deleting the lines would work fine on production, but it would screw up the test servers because of the checksum.